_posts/2024-03-14-multimodal-semantic-search.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: post
+title:  "Multimodal search: Searching with semantic and visual understanding"
+authors:
+  - gaievski
+  - haoyng
+  - dylantong
+date: 2024-03-24
+categories:
+  - technical-posts
+meta_keywords: Multimodal search, searching with semantic and visual understanding, improve search relevance, hybrid search in OpenSearch 2.10, keyword and image search
```

Comment:
@martin-gaievski - Please update the meta with the following: 

**Meta keywords:** neural query, multimodal search, search technology, search relevancy, OpenSearch 2.11

**Meta_description:** Learn about the enhancements to OpenSearch 2.11's neural query search by image capabilities, and dive deeper by exploring how multimodal models generate embeddings.


And please update the publish date to today, 4/11/24.

Reply:
@pajuric sure, just pushed new commit with requested changes